src/HasSlug.php

```diff
@@ -130,7 +134,19 @@ protected function makeSlugUnique(string $slug): string
         $originalSlug = $slug;
         $i = 1;
 
-        while ($this->otherRecordExistsWithSlug($slug) || $slug === '') {
+        while ($this->otherRecordExistsWithSlug($slug) || $slug === '' || in_array($slug, $this->slugOptions->forbiddenSlugs)) {
```

Comment:
This is unreadable. Could you extract this to a function with a good name? Also try to avoid `||` in there.